packages/rehype-katex/index.js

```diff
@@ -17,8 +17,11 @@ module.exports = function plugin (opts = {}) {
   if (opts.throwOnError == null) opts.throwOnError = false
   return function transform (node, file) {
     visit(node, 'element', function (element) {
-      const isInlineMath = element.tagName === 'span' && element.properties.className === 'inlineMath'
+      const isInlineMath = element.tagName === 'span' &&
+        element.properties.className &&
```

Comment:
This check seems necessary because `element.properties.className` can be undefined. This case is not covered by the tests though.